website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,192 @@
+## Usage
+
+### When to use
+
+- When the list of options is large and search is required to help filter the list.
+- When multiple selections are possible.
+- When list items are displayed in groups.
+- When you require complex structured content to be displayed within the list (and trigger).
```

Comment:
I _think_ we try to stay away from saying "you" and try to a bit more generic, but would love other opinions here. This isn't 100% right, but maybe moving in the right direction.
```suggestion
- When complex structured content is required to display the options within the list.
```

Reply:
I think this one needs a bit more thought. I'll look at it...

Reply:
Going to make this:
"- When complex, structured content is required to be displayed within the options."

(Feel free to suggest further changes for finessing)